entries/jQuery.speed.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.speed" return="PlainObject">
+  <title>jQuery.speed</title>
+  <desc>Creates an object containing a set of properties ready to be used in the definition of custom animations.</desc>
+  <signature>
+    <added>3.0.0</added>
```

Comment:
Looks like the three parameter version was added in 1.1 and a 2 parameter version existed in 1.0.